geometry2/d50d51b/d50d51b.bug

```diff
@@ -60,4 +60,4 @@ fault-codes:
   - PROGRAMMING:VALIDATION
 failure-codes:
   - SOFTWARE:BEHAVIORAL
-  - SYSTEM:MODEL-REALITY-DIVERGENCE
+  - SYSTEM:MONITORING
```

Comment:
I don't see the fit for monitoring here, either. The bug seems to be that static transforms aren't published with timestamps or frame identifiers. BEHAVING-INCORRECTLY feels more appropriate?

Reply:
From the meeting: let's do both MONITORING (due to frame identifiers) and BEHAVING-INCORRECTLY.